src/nvim/globals.h

```diff
@@ -140,7 +140,7 @@ EXTERN int mod_mask INIT(= 0x0);                /* current key modifiers */
  * temporarily moved.  The old position is restored with the next call to
  * update_screen().
  */
-EXTERN int cmdline_row;
+EXTERN long cmdline_row;
```

Comment:
Thank you, that was very helpful. Do you know if asserts can be optimized away in such cases?